### PR TITLE
Receive do documentation for Elixir

### PR DESCRIPTION
**Pull request by chriszimmerman**
I added some documentation on how the receive do block works for receiving messages in Elixir. I explained how a receive do block only waits for one message. In order to deal with multiple messages, a function with a receive do block must recursively call itself.

### CODE REVIEW

File: csharp.html.markdown [hunk 1]
```diff
@@ -394,7 +395,8 @@ on a new line! ""Wow!"", the masses cried";
             ref int maxCount, // Pass by reference
             out int count)
         {
-            count = 15; // out param must be assigned before control leaves the method
+			//the argument passed in as 'count' will hold the value of 15 outside of this function
```
Comment (vendethiel) [inline csharp.html.markdown]: seems like you messed up the indentation here

Comment (chriszimmerman) [inline csharp.html.markdown]: Thank you! I committed a fix for that issue.

File: elixir.html.markdown [hunk 2]
```diff
@@ -384,6 +391,8 @@ end
 
 # Compile the module and create a process that evaluates `area_loop` in the shell
 pid = spawn(fn -> Geometry.area_loop() end) #=> #PID<0.40.0>
+#Alternatively
```
Comment (vendethiel) [inline elixir.html.markdown]: please space this comment

Comment (chriszimmerman) [inline elixir.html.markdown]: Fixed

**Comment by adambard** [comment]
Looks good to me.

**Comment by vendethiel** [comment]
Thanks! :ship:

**Pull request merged**